the-exterminators/error-annihilator
# Sp 43 view dashboard

## Linked Issues

## PR Description
There are still Merge Conflicts, but it builds without problem on circle ci!
This view should only be a first version of the dashboard with a couple statistics. Please ignore my ridiculous comments, those are only a note for me to continue after the database implementation!

## Changes


## How to test

1. fetch branch
2. Open, build and run project
3. Check if project builds and looks correct on first sight
4. Navigate to dashboard
5. There should be a couple statistics with example in memory data

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
